rest-api/dao/hpo_dao.py

```diff
@@ -1,11 +1,31 @@
+from code_constants import UNSET
 from dao.cache_all_dao import CacheAllDao
+from dao.base_dao import FhirMixin, FhirProperty
+from dao.organization_dao import _FhirOrganization, OrganizationDao
 from model.hpo import HPO
+from model.organization import Organization
 from singletons import HPO_CACHE_INDEX
+from sqlalchemy.orm import subqueryload
+from fhirclient.models.domainresource import DomainResource
+
+class _FhirAwardee(FhirMixin, DomainResource):
+  """FHIR client definition of the expected JSON structure for an Awardee resource."""
+  resource_name = 'Awardee'
+  _PROPERTIES = [
+    FhirProperty('display_name', str, required=True),
+    FhirProperty('type', str, required=True),
+    FhirProperty('organizations', _FhirOrganization, is_list=True)
+  ]
+
+# Sort order for HPOs.
+_ORDER_BY_ENDING = ('name')
 
 class HPODao(CacheAllDao):
+
   def __init__(self):
     super(HPODao, self).__init__(HPO, cache_index=HPO_CACHE_INDEX,
-                                 cache_ttl_seconds=600, index_field_keys=['name'])
+                                 cache_ttl_seconds=600, index_field_keys=['name'],
+                                 order_by_ending=_ORDER_BY_ENDING)
```

Comment:
I guess I'm just confused by this parameter name, what is "ending' supposed to mean in this context?

Reply:
So in other APIs, we allow clients to specify a sort column in the request... when that is provided, the order_by_ending is effectively tacked on after that (excluding the requested sort column.)

In this API, we're only sorting by name, so it isn't relevant at the moment.